.github/workflows/publish_interpreter.yml

```diff
@@ -102,23 +102,28 @@ jobs:
       - name: Generate aquamarine.wasm.base64.js
         run: |
           cat << EOF > pkg/aquamarine.wasm.base64.js
-          export const wasmBs64 = "$(base64 -w0 pkg/aquamarine_client_bg.wasm)";
+          module.exports = "$(base64 -w0 pkg/aquamarine_client_bg.wasm)";
```

Comment:
if you know how it possible to use raw bytes instead of base64 string - it will be great

Reply:
I have some ideas how to improve it. Will do it a little bit later